src/utils/processContent.js

```diff
@@ -133,12 +142,21 @@ const renderScratchBlocks = (content, styles) => {
     if (blocks) {
       blocks.forEach(block => {
         let code = block.substring(r.start.length, block.length - r.end.length);
-        let SVG = scratchblocks(code, r.options);
-        returnContent = returnContent.replace(block, SVG);
+        let doc = scratchblocks.parse(code, r.options);
+        let docView = scratchblocks.newView(doc, {style: 'scratch3'});
+        let svg = docView.render();
+        if (r.options.inline) {
+          svg.style.margin = '3px 0';
+          svg.style.verticalAlign = 'middle';
+        }
+        else {
+          svg.style.display = 'block';
+          svg.style.margin = '0 auto 15px';
```

Comment:
Kanskje heller `0 0 15px` slik at scratchblokkene kommer til venstre i stedet for å bli midtstilt? Eller har dette blitt diskutert? @tjesi , @gahjelle ?